### PR TITLE
Adding a approving leave guide for management

### PR DESCRIPTION
**Pull request by bvhme**
Reflect approving leave as a process and update the leave guide to be in line with it.

-----
[View rendered activities/staff-information/leave.md](https://github.com/publiccodenet/about/blob/leave-management/activities/staff-information/leave.md)
[View rendered activities/staff-management/approving-leave.md](https://github.com/publiccodenet/about/blob/leave-management/activities/staff-management/approving-leave.md)

### CODE REVIEW

File: activities/staff-management/approving-leave.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+type: Resource
+---
+
+# Approving leave
+
+How to approve leave requests for vacation, not for sick, pregnancy, care or emergency leave.
+
+A leave request email is submitted through the personal Tentoo account of the employee or through email if employed outside of the Netherlands. in line with the [staff leave guide](../staff-information/leave.md).
```
Comment (clausmullie) [inline activities/staff-management/approving-leave.md:9]:
```suggestion
A leave request email is submitted through the personal Tentoo account of the employee or through email if employed outside of the Netherlands, in line with the [staff leave guide](../staff-information/leave.md).
```

File: activities/staff-management/approving-leave.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+1. See the leave overview in Tentoo NMBRS to see if their leave overlaps with their colleagues and if necessary ask if they have discussed with their planning with their team.
```
Comment (clausmullie) [inline activities/staff-management/approving-leave.md:11]:
```suggestion
1. Check the leave overview in Tentoo NMBRS to see if their leave overlaps with their colleagues, and if necessary ask if they have discussed their planning with their team.
```

File: activities/staff-management/approving-leave.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+2. Check the balance of leave in the leave overview. If this balance is low offer for them to buy their leave.
```
Comment (clausmullie) [inline activities/staff-management/approving-leave.md:12]:
```suggestion
2. Check the balance of leave in the leave overview. If this balance is low, remind them of the possibility to buy extra leave.
```

**Review by clausmullie — CHANGES_REQUESTED** [review]
Some in-line suggestions above

**Review by ericherman — CHANGES_REQUESTED** [review]
Please commit the suggested commas from Claus

**Review by ericherman — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

**Review by clausmullie — APPROVED** [review]

**Pull request merged**